src/components/rating/Rating.tsx

```diff
@@ -83,6 +83,7 @@ class Rating extends React.Component<RatingPropsType> {
     (arg0: React.MouseEvent<HTMLSpanElement>) => unknown
   > = [];
 
+  // It's a typo, should be componentWillReceiveProps (UNSAFE_componentWillReceiveProps)
```

Comment:
can we fix this? what is the solution here?

Reply:
The solution would be to use `UNSAFE_componentWillReceiveProps` but this will change behaviour of the component as the previous `componentWillReciveProps` was never called.
Other solution would be to remove it completely